3/introduction.md

```diff
@@ -1,7 +1,18 @@
-## Motivation for security
-* currently our communication system uses plaintext (https://github.com/libp2p/specs/tree/master/plaintext) which is not secure. Use a alice bob diagram
-* Explain types of encryption in connection, a breif about symmetric and asymmetric encryption with nice diagrams and good resources 
-    - very detailed lectrue series (https://www.youtube.com/channel/UC1usFRN4LCMcfIV7UjHNuQg/featured)
-    - computerphile https://www.youtube.com/watch?v=NmM9HA2MQGI
-* common practices in connection encryption
-* Lets add the security layer to our chat system
+## Secure Communication
+
+As has been noted several times throughout this tutorial, libp2p is a modular network stack and one of the module types available for libp2p is the [connection encryption](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#connection-encryption) module type, which is set using the `connEncryption` option when configuring a node.  As the name suggests, this module deals with how data that is sent between two nodes is encrypted using asymmetric encryption.  
```

Comment:
```suggestion
Libp2p has [connection encryption](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#connection-encryption) interface to easily plug use any encryption method.  This can be configured by setting the `connEncryption` in node options.  This will make sure every transport message is encrypted using the provide method.   
```